site/content/en/docs/Advanced/multi-cluster-allocation.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Multi-cluster Allocation"
+date: 2019-10-25T05:45:05Z
+description: >
+  In order to allow allocation from multiple clusters, Agones provides a mechanism to set redirect rules for allocation requests to the right cluster.
+---
+
+There may be different types of clusters, such as on-premise, and Google Kubernetes Engine (GKE), used for a game to help with the cost saving and availability. To plan for the cluster usage and boost availability, Agones provides a mechanism to define priorities on the clusters. Priorities are defined on [GameServerAllocationPolicy](https://github.com/googleforgames/agones/blob/master/pkg/apis/multicluster/v1alpha1/gameserverallocationpolicy.go) agones CRD. A match maker can enable the multi-cluster rules on a request and target [agones-allocator]({{< relref "allocator-service.md">}}) endpoint in any of the clusters and get resources allocated on the cluster with the highest priority. If the cluster with highest priority is overloaded, the allocation request is redirected to the cluster with the next highest priority.
```

Comment:
nit: I'd remove the bit about cost savings and just focus on availability (maybe break it into reliability -- e.g. uptime availability -- and locality -- e.g. latency to users).